src/lib.rs

```diff
@@ -36,21 +38,23 @@ pub enum LndConnectError {
 impl Lnd {
     pub async fn connect<D>(
         destination: D,
-        certificate_bytes: &[u8],
-        macaroon_bytes: &[u8],
+        certificate_path: &str,
+        macaroon_path: &str,
```

Comment:
Isn't this something that is very particular to how we do things? Other users might not be reading it from a path, but might already have the certificate and macaroon bytes. Should we care about that? Or is this a future problem we can quickly solve by adding an extra `connect` function? 🤔